h2o-py/tests/testdir_misc/pyunit_uniop_colname_domain.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+#Test to see if you apply H2O uniop methods to an H2OFrame it will not overwrite the original column headers & domains
+# & also update colnames of new frame with `op(colName)` where `op` is a uniop operator.
+
+def pyunit_colname_uniop():
+    dataframe = {'A': [1,0,3,4], 'B': [5,6,-6, -1], 'C':[-4, -6, -7, 8]}
+    frame = h2o.H2OFrame(dataframe)
+    frame_asin = frame.asin()
+
+    #Check colnames of original frame remain untouched.
+    assert frame.names == [u'A', u'C', u'B'], "Expected original colnames to remain the same after uniop operation"
```

Comment:
First, you don't have to add prefix `u` in front of the strings that don't contain any unicode in them.
Second, this test is wrong. When you create a frame from a dictionary, the vecs are added in non-deterministic quasi-random order. I don't know the exact details, but it's definitely bad to assume that the order will end up being "A", "C", "B" always. Instead, you'd want to write 
```
assert set(frame.names) == {"A", "B", "C"}
```